src/elasticsearch/user-elasticsearch.service.ts

```
@@ -206,8 +227,24 @@ export class UserElasticsearchService implements OnModuleInit {
     fetchUserFromDb?: (userId: string) => Promise<IUser | null>
   ): Promise<any> {
     try {
-      // Try to update the profile field in Elasticsearch
-      const normalizedProfile = this.normalizeProfileForElasticsearch(profile);
+      const existing = await this.getUser(userId);
+      let profileToNormalize = profile;
+      if (
+        existing?._source &&
+        profile &&
+        typeof profile === 'object' &&
```

ESLint

[error] Replace `'object'` with `"object"`

(prettier/prettier)